### ISSUE TITLE
NTR: History of TPN dependence (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7852]
New term request History of TPN dependence (brumme_220426182923): Total parenteral nutrition, includes malabsorption syndromes, malnutrition, nutritional deficiency, often requires central venous access, PICC placement  UMLS: 4069271

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000169
- label: History of total parenteral nutrition dependence
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: PMID:32644462
- def: A past medical history of a recent depence on total parenteral nutrition (TPN), which refers to intravenously administered nutrition being the only source of nutrition the patient is receiving.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7852
- synonyms:
